lansweeper_app_for_splunk/default/data/ui/views/ls_assets.xml

```diff
@@ -1,7 +1,7 @@
 <form>
-  <label>Assets</label>
+  <label>Assets Clone</label>
   <fieldset submitButton="false">
-    <input type="dropdown" token="tkn_site">
+    <input type="dropdown" token="tkn_site" depends="$hide$">
```

Comment:
Let's remove it completely, as we are already removing its usage. If we need it in the future we can add it again.

Reply:
Removed Input